### PR TITLE
Update mac installer link for 2.0.4/1473

### PR DESCRIPTION
**Pull request by steveatinfincia**
Note: depends on `.sig` file at Github:

`https://github.com/freenet/mactray/releases/download/v2.0.4/FreenetTray_2.0.4.zip.sig`

being pushed to: 

`https://downloads.freenetproject.org/alpha/mactray/FreenetTray_2.0.4.zip.sig`

### CODE REVIEW

File: pages/download.py [hunk 1]
```diff
@@ -133,8 +133,8 @@ def get_content(self):
 
 Freenet requires OS X 10.8 or later.
 """) + "\n\n" + """
-[url_mac_installer]: https://github.com/freenet/mactray/releases/download/v2.0.3/FreenetTray_2.0.3.zip
-[url_gpg_sig]: https://downloads.freenetproject.org/alpha/mactray/FreenetTray_2.0.3.zip.sig
+[url_mac_installer]: https://github.com/freenet/mactray/releases/download/v2.0.4/FreenetTray_2.0.4.zip
+[url_gpg_sig]: https://downloads.freenetproject.org/alpha/mactray/FreenetTray_2.0.4.zip.sig
```
Comment (nextgens) [inline pages/download.py:137]: Why aren't you linking to github there too?
IMHO we should either mirror both or none

Comment (steveatinfincia) [inline pages/download.py:137]: I think @Thynix goal was to offload the rather large download to GitHub but still keep the `.sig` on the server to have a bit more control over what was signed/linked from the website, not sure.

Comment (nextgens) [inline pages/download.py:137]: That made sense for the .sha1 files... but not for the GPG signatures; let's keep it simple and serve it from the same server (github in this instance)

**Comment by Thynix** [comment]
Deployed, thanks!

**Pull request merged**